prereise/gather/helpers.py

```diff
@@ -46,10 +47,9 @@ def get_monthly_net_generation(state, eia_form_923, resource, hps=True):
     if not isinstance(resource, str):
         raise TypeError("resource must be a str")
 
-    if state not in abv2loadzone.keys():
+    if state not in abv2state.keys():
```

Comment:
You could optionally remove the `.keys()` here and in the `ValueError` (the `in` operator checks keys, and calling `set` on a dict gives back the keys). Not sure if that's less readable though.

Reply:
Good catch, done.